### ISSUE TITLE
Repository rename breaks existing configuration

### ISSUE THREAD

**Issue by mlwarner** [Pocco81/true-zen.nvim#83]
Hey there!

It looks like the repository was recently renamed from `Pocco81/TrueZen.nvim` -> `Pocco81/true-zen.nvim` which breaks some setups.

I discovered this through my nix setup which relies on [Pocco81/TrueZen.nvim](https://github.com/NixOS/nixpkgs/blob/634141959076a8ab69ca2cca0f266852256d79ee/pkgs/applications/editors/vim/plugins/vim-plugin-names#L636) that no longer exists in nix nightly but I'm anticipating that others will stop seeing updates for other packaging systems as well e.g. [`Plug 'Pocco81/TrueZen.nvim'`](https://github.com/Pocco81/true-zen.nvim#vim-plug).

Should users update to the new naming? And if so, could we update the installation guide? Thanks!

**Comment by Pocco81** [comment]
Hey @mlwarner :) I disliked a little the code base of the plugin so I decided to rewrite it. Currently everything is under the `dev` branch but the update will eventually reach main (probably within the next couple of minutes).

Be prepared though, everything will break! I'm not going to offer backwards compatibility with existing truezen configs, however I'll **make a migration guide** as soon as possible.

Don't worry! this new update is way way better and is 100% lua :tada:

See: https://github.com/Pocco81/true-zen.nvim/tree/dev
